forge/src/main/java/org/openjdk/skara/forge/github/GitHubPullRequest.java

```diff
@@ -703,6 +703,20 @@ public void makeNotDraft() {
             .execute();
     }
 
+    @Override
+    public Optional<ZonedDateTime> lastMarkedAsDraftTime() {
+        var lastMarkedAsDraftTime = request.get("issues/" + json.get("number").toString() + "/timeline")                .execute().stream()
+                .map(JSONValue::asObject)
+                .filter(obj -> obj.contains("event"))
+                .filter(obj -> obj.get("event").asString().equals("convert_to_draft"))
+                .reduce((a, b) -> b)
```

Comment:
Are you sure of the order events are returned in? I couldn't find a definition of the order. Maybe safer to map to date and reduce to the highest one?

Reply:
I can't find the related document too. I tried the rest api to get the related data and found the order personally. Now I adjust the code to find the highest one to avoid the API result changes in the future.